README.md

```diff
@@ -64,6 +64,8 @@ CHANGELIST
 - Added user sort to full meeting list editors on service body administration.
 - Fix for default duration time not displaying correctly on meeting editor.
 - Added sort to meeting format checkboxes on formats page by format code.
+- Made initialize database button easier to see on installer wizard, and settings only gets displayed once button is mashed.
+- Added cache busting to installer wizard.
```

Comment:
Will you be more specific about the files you add cache busting to? Like, say you added cache busting to the installer wizard's javascript and css files.